src/main/java/org/jboss/forge/arquillian/command/AddContainerDependencyStep.java

```diff
@@ -96,18 +99,33 @@ public Result execute(UIExecutionContext context) throws Exception {
       Map<Object, Object> ctx = context.getUIContext().getAttributeMap();
       Container container = (Container)ctx.get(ContainerSetupWizard.CTX_CONTAINER);
       String version = (String)ctx.get(ContainerSetupWizard.CTX_CONTAINER_VERSION);
+      Project project = getSelectedProject(context);
+      ArquillianFacet arquillian = project.getFacet(ArquillianFacet.class);
+      ArquillianConfig config = arquillian.getConfig();
 
-      containerInstaller.installContainer(
-            getSelectedProject(context),
-            container,
-            version,
-            getVersionedDependenciesMap());
+      final String profileId = container.getProfileId();
+
+      if (container. isSupportedByChameleon(version)) {
+         dependencyManager.addChameleonDependency(project);
+         if (config.isContainerWithDefault("true")) {
+            config.addContainer(profileId);
+         } else {
+            config.addContainerWithAttribute(profileId, "default", "true");
+         }
```

Comment:
Ok I think I understand what this code is supposed to do. I think we might improve its readability to avoid similar confusion for other contributors.

How about `if (config.isContainerWithDefault("true"))` -> `if (config.containsDefaultContainer())` .

I understand that rationale behind this check is to add default container if and only if there is no other already defined like that. Maybe my proposed method name reveals that intent a bit more?